src/main/resources/resources/logisim/strings/std/std_pl.properties

```diff
@@ -798,6 +798,7 @@ TTL7432 = 7432: poczwórna bramka 2-wejściowa OR
 # ==> TTL7434 =
 TTL7436 = 7436: poczwórna bramka 2-wejściowa NOR
 TTL74377 = 74377: ośmiokątny D-Flipflop z zezwoleniem
+# ==> TTL74381 =
```

Comment:
Either

    TTL74381 = Jednostka arytmetyczno-logiczna

or

    TTL74381 = Jednostka arytmetyczno-logiczna (ALU)

Reply:
Done

Thanks for the translation !